.github/workflows/black.yml

```diff
@@ -0,0 +1,18 @@
+name: Black format check
+
+on:
+  push:
+    branches:
+    - main
+    
+jobs:
+  lint:
+    name: lint
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - name: Black Code Formatter Check
+        uses: RojerGS/python-black-check@master
+        with:
+          line-length: '88'
```

Comment:
Can we set this to something wider?  Maybe 120?  flake8 max line length is 150.

Reply:
We would need to change the config in the pre-commit.yml as well. By default it is 88 afaik.

Reply:
Yes that is correct the default is 88 and that's the default for black formatting also. And fyi the reason I had mentioned it to 88 explicitly in the yaml file is because the github action that I am using has default set to 81 for some reason 🤷